src/activities/webhooks/Elsa.Activities.Webhooks/Extensions/WebhookOptionsBuilderExtensions.cs

```diff
@@ -13,7 +13,8 @@ public static ElsaOptionsBuilder AddWebhooks(this ElsaOptionsBuilder elsaOptions
             elsaOptions.Services
                 .AddScoped<IActivityTypeProvider, WebhookActivityTypeProvider>()
                 .AddBookmarkProvider<WebhookBookmarkProvider>()
-                .AddNotificationHandlersFrom<EvictWorkflowRegistryCacheHandler>();
+                .AddNotificationHandlersFrom<EvictWorkflowRegistrySavedCacheHandler>()
+                .AddNotificationHandlersFrom<EvictWorkflowRegistryDeletedCacheHandler>();
```

Comment:
You can remove these lines - the handlers will all be registered automatically because of `AddNotificationHandlersFrom<EvictWorkflowRegistryCacheHandler>()`.